react/future/react-0.13.0.d.ts

```diff
@@ -118,19 +118,19 @@ declare module "react" {
     // ----------------------------------------------------------------------
 
     // Base component for plain JS classes
-    class Component<P, S, C> implements ComponentLifecycle<P, S, C> {
-        constructor(props: P, context: C);
+    class Component<P, S> implements ComponentLifecycle<P, S> {
+        constructor(props: P, context: any);
         setState(state: S, callback?: () => any): void;
         forceUpdate(): void;
         props: P;
         state: S;
-        context: C;
+        context: any;
```

Comment:
Given the comments in the last thread, it seems that the context will always be an Object? I guess any is safer though